packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -72,7 +72,7 @@ export class DocumentDeltaConnection
 
 	private socketConnectionTimeout: ReturnType<typeof setTimeout> | undefined;
 
-	private _details: IConnected | undefined;
+	protected _details: IConnected | undefined;
```

Comment:
Add a comment to this?  Esp curious when it's undefined v. defined.

Also, not sure if `_` prefix is usually for private-only, or also protected.  We don't have any repo guidelines on it AFAIK, so not critical for sure.

Reply:
Should be able to remove it if not required in ODSPDocumentDeltaConnection.